stellarphot/settings/models.py

```diff
@@ -617,6 +617,15 @@ class PhotometryOptionalSettings(BaseModelWithTableRep):
         the star. If ``False``, the FWHM will be calculated by finding the
         second order moments of the light distribution. Default is ``True``.
 
+    method : `typing.Literal["exact", "center", "subpixel"]`, optional
```

Comment:
If this is optional, I think you want to state 'exact' is the default if it is unset as part of the documentation.